packages/components/src/button/Button.module.css

```
@@ -33,7 +33,7 @@
     background-color: var(--midas-button-background-primary-hover);
   }
 
-  &[data-pressed] {
+  &[data-pressed]:not([aria-expanded='true']) {
```

```suggestion
  &[data-pressed]:not([aria-expanded='true']):not([aria-haspopup='true']) {
```

Sådär skulle du ju kunna göra för att vara lite extra specifik?